libs/portals/form-system/ui-components/src/components/FieldComponents/List/List.tsx

```
@@ -1,7 +1,9 @@
 import { FormSystemField, FormSystemListItem } from '@island.is/api/schema'
-import { Dispatch } from 'react'
+import { Dispatch, useEffect } from 'react'
 import { Select } from '@island.is/island-ui/core'
 import { Action } from '../../../lib/reducerTypes'
+import { getValue } from '../../../lib/getValue'
+import { get } from 'lodash'
```

ESLint

[error] 'lodash' import is restricted from being used.

(no-restricted-imports)